examples/layer-browser/data/sf.trips.json

```diff
@@ -0,0 +1,101 @@
+[
+  {
+    "segments": [
+      [
+        -122.39079879999997,
+        37.7664413,
+        0
```

Comment:
Can we use real timestamps for this sample? It has been a confusing point to our users.